guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,37 @@
+[id="security-features_{context}"]
+= Security features in {Project}
```

Comment:
```suggestion
= Security hardening in {Project}
```
This seems to be the standard term used in RHEL docs.

Reply:
It is an established term indeed, but I don't think it would be a good heading here. The module is a conceptual/referential list of the available security features, rather than a specific guide how to set them up them. Also, using a gerund heading for a non-procedural module should be avoided according to our style guides.

Reply:
I'd agree it's not a good heading. Many of these are more compliance than hardening. For example, FIPS only has a certainly list of allowed ciphers. Newer ciphers (which are arguably better) may not be on the list until a new standard is allowed. Effectively FIPS can make a system less secure, but compliant. An example we had in the past: FIPS didn't allow the use of MD5, but RHEL 6 only had MD5 checksums for RPMs. The solution: disable checksums altogether.

IMHO nobody should use FIPS unless you're mandated to.